frontend/src/api.js

```diff
@@ -161,6 +161,12 @@ class Api {
 
   //----TEAM STATS---
 
+  // @Ellington pretend this "TEAM STATS" section doesn't exist for now...
+  // any calls made here, ignore that for now, treat that as part 2
+  // (and we can help you when you get there)
+  // Some of these old API calls have direct translations to new ones
+  // Some of them don't (ie they were backend methods that no longer exist anymore) so...
```

Comment:
can you change this comment to something like "clean these calls, fix in #368"?
and ofc remove your name since it doesn't have to be you